.travis.yml

```diff
@@ -43,16 +43,23 @@ matrix:
         apt:
           packages:
           - language-pack-zh-hans
+
     - dist: trusty
       env:
         - JOB="2.7" ENV_FILE="ci/deps/travis-27.yaml" PATTERN="not slow"
       addons:
         apt:
           packages:
           - python-gtk2
+
```

Comment:
can you also add this in the allowed failures section, i am a little leary of adding a 6th job

Reply:
I'm planning to move the jobs (or some of them) not using mysql or postgres to azure after this PR.

I can surely move it to allowed failures, but we shouldn't have 6 jobs for more than couple of days, if that makes a difference.

Reply:
ok that’s fine the

Reply:
The CI is green here, I expected circle to not run, as it's been removed from the repo. But it just looks as queued. I guess it won't appear when the config is deleted from master, or we may have to unlink our disable our account or something.